src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        Supplier<RuntimeException> exceptionSupplier = 
+                () -> new RuntimeException("Can't get min value from list: " + numbers.toString());
+        int minInt = numbers.stream()
```

Comment:
do not create single use variables
```suggestion
        return numbers.stream()
```